topo_processor/util/conversions.py

```diff
@@ -71,3 +48,22 @@ def nzt_datetime_to_utc_datetime(date: str) -> datetime:
     utc_time = nz_time.astimezone(utc_tz)
 
     return utc_time
+
+
+def quarterdate_to_date_string(value: str) -> str:
```

Comment:
That looks better to me now :) thanks
I would suggest create a UT for it too

Reply:
I've just done that and pushed.